src/Apps/Artwork/Components/ArtworkDetails/index.tsx

```diff
@@ -36,3 +39,45 @@ export const ArtworkDetailsFragmentContainer = createFragmentContainer(
     }
   `
 )
+
+export const ArtworkDetailsQueryRenderer = ({
+  artwork,
+}: {
+  artwork: string
+}) => {
```

Comment:
This can be typed to an `SFC<ArtworkDetailsProps>`

Reply:
I think we want to just pass `id` for artwork here and `SFC<ArtworkDetailsProps>` would expect artwork as a collection of attributes already, no? Maybe I should not name it `artwork` but rather `artworkId` here?

Or am I missing something here @damassi?